packages/node_modules/samples/browser-plugin-meetings/PROPOSAL.md

```diff
@@ -0,0 +1,77 @@
+# Meetings Plugin Kitchen Sink Reorganization Proposal
+
+## General Styling and Helper Queues
+
+- Each Logic Group Should be Collapsable / Disableable
+- The Presentational Order Should Match Group Order in this Document
+- Proper Presentation on Provided Listable Values Should be Implemented
+
+## Logic Grouping
+
+### Authentication
+
+- Allow User Token Input
+- Enables All Other Logic Groups Upon Successful Authentication
+
```

Comment:
Let's add a line about calling "register" instead of "connect". `meetings.register()` is a function, `meetings.connect` isn't and might cause some confusion. Also need to add `meetings.unregister`

Reply:
Added notes around this.